libparsec/crates/platform_pki/examples/decrypt_message.rs

```diff
@@ -39,11 +44,13 @@ fn main() -> anyhow::Result<()> {
         .decode(&b64_data)
         .context("Failed to decode hex encoded data")?;
 
-    let res = decrypt_message(&data, &cert_ref).context("Failed to decrypt message")?;
+    let res =
+        decrypt_message(args.algorithm, &data, &cert_ref).context("Failed to decrypt message")?;
 
     println!(
-        "Decrypted by cert with id: {}",
-        data_encoding::BASE64.encode_display(&res.cert_ref.id)
+        "Decrypted by cert with id {{{}}} with algo {}",
```

Comment:
`{{{}}}` ? o_O'

Reply:
`{{` and `}}` is the syntax to espace `{` and `}` respectivelly